bios/team-three/ben-wagner.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lab 1 - Ben Wagner</title>
+    </head>
+    <body>
+        <h1>CSCI 2479 Spring 2022 &mdash; Ben Wagner Bio</h1>
+        <nav>
+          <div><a href="./index.html">Back to Team three</a></div>
+        </nav>
+        <div class="bio">
+            <h2>Ben Wagner</h2>
+            <ul class="qa">
+              <li><b>Where you are from originally?</b><br> Columbus, Ohio</li><br>
+              <li><b>Your major/track at CSCC?</b> <br> Web Development AAS</li><br>
+              <li>
+                <b> What you hope to do after school?</b> <br> I would like to find a relatively non-stressful web development job. Meaning a job that is within my capabilties yet I am still able to grow.
```

Comment:
This is a good thing to look for. Remote might be of scope within what you are looking for.